docs/templates/base.html

```diff
@@ -0,0 +1,140 @@
+{% extends "adidoks/templates/base.html" %}
+{% block footer %}
+{{ super() }}
+<script>
+document.addEventListener("DOMContentLoaded", inject);
```

Comment:
Can we pull this out into a separate .js file? It looks like Zola supports this eg. [here](https://github.com/getzola/zola/blob/master/docs/templates/index.html#L107).

Reply:
Fixed